addons/account/static/src/scss/account_journal_dashboard.scss

```diff
@@ -64,3 +64,8 @@
     fill: gray !important;
     opacity: 0.1;
 }
+
+/* Prevent o_field_kanban_vat_activity entries from spanning entire screen width. */
+.o_kanban_card_content > .o_field_widget.o_readonly_modifier.o_field_kanban_vat_activity {
+    display: inline
```

Comment:
looks good to me

if it works I would prefer `display: block` since it's the default value that is overridden by the o_field_widget CSS:

https://github.com/odoo/odoo/blob/82b03c37327abbab0b0bb63ba2423fe475004b66/addons/web/static/src/legacy/scss/fields.scss#L71

(I guess the issue happen because we combine inline-block as a flex item)

also I think the selector could probably be simplified a little, and added inside the:

```py
.o_kanban_dashboard.o_account_kanban {
    .o_kanban_record {
```
IMO just a select `.o_field_widget.o_field_kanban_vat_activity` inside that block should be enough

Reply:
Done!